notebooks-html/bias_modeling.html

```diff
@@ -77,38 +77,39 @@
 \end{aligned}
-$$<p>We denote the process that generates the true population data as $ f_\theta $. Our observed outcomes $ y $ were generated by our population process plus some random noise in data collection: $ y_i = f_\theta(X_i) + \epsilon $.</p></div></div></div>
+$$</p>
+<p>We denote the process that generates the true population data as $ f_\theta $. Our observed outcomes $ y $ were generated by our population process plus some random noise in data collection: $ y_i = f_\theta(X_i) + \epsilon $, where the random noise $ \epsilon $ is a random variable with a mean of zero: $ \mathbb{E}[\epsilon] = 0 $.</p></div></div></div>
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
 <p>If we use the squared error as our loss function, the above expression becomes:</p>
-$$
+<p>$$
 \begin{aligned}
 R(f_\hat{\theta}) = \mathbb{E}[ (\gamma - f_\hat{\theta} (z))^2 ]
 \end{aligned}
-$$</div></div></div>
+$$</p></div></div></div>
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
 <p>With some algebraic manipulation that we omit for brevity, we can show that the above expression is equivalent to:</p>
-$$
+<p>$$
 \begin{aligned}
 R(f_\hat{\theta}) = (\mathbb{E}[f_\hat{\theta}(z)] - f_\theta(z))^2 + \text{Var}(f_\hat{\theta}(z)) + \text{Var}(\epsilon)
 \end{aligned}
-$$</div></div></div>
+$$</p></div></div></div>
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<p>The first term in this expression, $ (\mathbb{E}[f_\hat{\theta}(z)] - f_\theta(z))^2 $, is a mathematical expression for the bias of the model. The bias is equal to zero if in the long run our choice of model $ f_\hat{\theta}(z) $ predicts the same outcomes produced by the population process $ f_\theta(z) $. The bias is high if our choice of model makes poor predictions of the population process even when we have the entire population as our dataset.</p>
-<p>The second term in this expression, $ \text{Var}(f_\hat{\theta}(z)) $, represents the model variance. The variance is low when the model's predictions don't change much when the model is trained on different datasets from the population. The variance in high when the model's predictions change greatly when the model is trained on different datasets from the population.</p>
+<p>The first term in this expression, $ (\mathbb{E}[f_\hat{\theta}(z)] - f_\theta(z))^2 $, is a mathematical expression for the squared bias, $\text{bias}^2$, of the model. The bias is equal to zero if in the long run our choice of model $ f_\hat{\theta}(z) $ predicts the same outcomes produced by the population process $ f_\theta(z) $. The bias is high if our choice of model makes poor predictions of the population process even when we have the entire population as our dataset.</p>
```

Comment:
On a second read, I remember not stating that this term was the squared bias because I didn't think it was an important point and might be a bit confusing for people seeing bias for the first time. Do you think you can reword this to something like:

> is a mathematical expression for the bias of the model. (Technically, this term represents the squared bias.)